scripts/build_test.py

```diff
@@ -28,22 +29,144 @@
 
 # pylint: enable=relative-import
 
+
+TEST_DIRECTORY = os.path.join('core', 'tests', 'build', '')
```

Comment:
Here and below, do you need the last `''`? First time I've seen that being needed, so just wondering.